org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizardPage.java

```diff
@@ -217,15 +226,67 @@ public void createControl(Composite parent) {
 		templateViewer = new ListViewer(list);
 		templateViewer.setContentProvider(new ArrayContentProvider());
 		templateViewer.setComparator(new ViewerComparator()); // default uses getLabel()/toString()
-		templateViewer.add("Loading templates");
-		templateViewer.getList().setEnabled(false);
 		templateViewer.addSelectionChangedListener(e -> {
 			setPageComplete(isPageComplete());
 		});
 		templateControlDecoration = new ControlDecoration(templateViewer.getControl(), SWT.TOP | SWT.LEFT);
 		templateControlDecoration.setImage(errorImage);
+		updateTemplateList();
+
+		//Update Template List with preferences change
+		IPreferenceStore store = AcutePlugin.getDefault().getPreferenceStore();
+		updateTemplatesListener = new IPropertyChangeListener() {
+			@Override
+			public void propertyChange(PropertyChangeEvent event) {
+				if (event.getProperty().equals(AcutePreferenceInitializer.explicitDotnetPathPreference)) {
+					updateTemplateList();
+				}
+			}
+		};
+		store.addPropertyChangeListener(updateTemplatesListener);
+
+		new Label(container, SWT.NONE);
+
+		new Label(container, SWT.NONE);
+		Link preferencesLink = new Link(container, SWT.NONE);
+		preferencesLink.setText("<a>.NET Preferences</a>");
+		preferencesLink.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false,2,1));
+		preferencesLink.addSelectionListener(widgetSelectedAdapter(s -> Display.getDefault().asyncExec(() -> {
+			PreferenceDialog preferenceDialog = PreferencesUtil.createPreferenceDialogOn(getShell(),
+					AcutePreferencePage.PAGE_ID,
+					new String[] { AcutePreferencePage.PAGE_ID }, null);
+			preferenceDialog.setBlockOnOpen(false);
+			preferenceDialog.open();
+		})));
+		new Label(container, SWT.NONE);
+
+		Composite workingSetComposite = new Composite(container, SWT.NONE);
+		GridData layoutData = new GridData(SWT.FILL, SWT.FILL, true, false, 4, 1);
```

Comment:
The veritcal spacing here is too small.
I have the impression that vertical spacing between project name and template list is bigger than the one between template list and working set. I imagine it'd be better to have it the same typical height of a Label widget I believe). You should probably add a label spacer before the workingSetComposite, somthing like `new Label(container, SWT.NONE).setLayoutData(new GridData(SWT.FILL, SWT.DEFAULT, true, false, 4, 1)`.